src/components/Tabs/Tabs.vue

```diff
@@ -9,12 +9,11 @@
 </style>
 
 <script>
-import iamTabs from '../../../assets/js/components/tabs/tabs.component.min.js'
+import iamTabs from '../../../assets/js/components/tabs/tabs.component.js'
```

Comment:
Can we put the .min back please. I know this is annoying and ill try to come up with dev/production flag or something in the future.